org-formation/090-systems-manager/QuickSetup-PatchPolicy.yaml

```diff
@@ -0,0 +1,92 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: An SSM patch policy to patch all instances in the given OUs
```

Comment:
there's no ability to target an OU is there?  `TargetOrgRoot` means the entire organization, correct?